src/camera/pi/meowlpi/camera/streamer.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+import os
+import settings
+
+class MeowlPiStreamer():
+    """
+        A ffmpeg wrapper to convert the stream in flv and send it to server
+    """
+    def __init__(self):
+        """Opens a new process to run ffmpeg"""
+        self.__ffmpeg_command = [
+            'ffmpeg', '-y', 
+            '-f', settings.STREAM_INPUT_FORMAT,
+            '-i', '-', '-c:v', 'copy', '-map',
+            '0:0', '-f', settings.STREAM_OUTPUT_FORMAT, 
+            os.environ['SERVER_RTMP_URL_LINK']]
```

Comment:
Maybe add a check in create_app to see if `SERVER_RTMP_URL_LINK` exists otherwise this throws an exception

Reply:
I exported all of these environment variables in ./run before running the application. The chances that it doesn't see it is none.

Reply:
Would you still need  `if os.environ.get('RASPBERRY_PI_FLASK_SECRET_KEY') is None:` ?

Reply:
I decoupled the environment variables and move them to `settings.py`. In there, the stream will redirect to /tmp/video.flv if it doesn't exists. I expect Kubernetes to maintain these environment variables in the future.